roadmap.md

```diff
@@ -0,0 +1,14 @@
+# Roadmap
+
+**Deliverable**|**Status**|**Expected Delivery**|**Description**
+:-----:|:-----:|:-----:|:-----:
+Broker In Browser |In Progress|TBD|Application model for application nested in iframes, to support silent token acquisition where 3p cookies are blocked. 
+Msal Node Public Preview|In Progress|October 2020|Our Msal Node library is currently in private preview.  Public preview should include full Public Client and Confidential Client support.  When released, we will be committing to supporting this library at a production level, resolving bugs in a timely manner. 
+SN + I Support In Msal Node |Not Started|December 2020|Explanation of the flow from our python library: https://github.com/AzureAD/microsoft-authentication-library-for-python/issues/60
+Refresh Token Proof of Possession [Browser]|Not started|December 2020|Proof of Possession is a feature that enables cryptographically locking a token to a browser.  This feature has already been shipped for Access Tokens.
+Tenant Profile Support |Not Started|December 2020|Support for tenant profiles in our cache and account management.
+Msal Angular 2.0 [Support for Auth Code Flow]|In Progress|December 2020|Upgrading our Msal Angular library to use the Msal Browser lib we recently GA'd to support the Auth Code Flow in the browser.  Helps alleviate some issues caused by blocked 3p cookies. 
```

Comment:
Can we split this into public/private preview?

Reply:
I am curious what difference we see. Honestly my hope is that we can ship the whole thing at the end of the quarter and potential have some betas ready to test before hand.

Reply:
I still think we should split it to set customer expectations!